deployment/roles/ui/templates/.env.j2

```diff
@@ -0,0 +1,34 @@
+REACT_APP_HOME_BRIDGE_ADDRESS={{ HOME_BRIDGE_ADDRESS }}
+REACT_APP_FOREIGN_BRIDGE_ADDRESS={{ FOREIGN_BRIDGE_ADDRESS }}
+REACT_APP_FOREIGN_HTTP_PARITY_URL={{ FOREIGN_RPC_URL }}
+REACT_APP_HOME_HTTP_PARITY_URL={{ HOME_RPC_URL }}
+
+REACT_APP_HOME_NATIVE_NAME={{ HOME_NATIVE_NAME }}
+
+REACT_APP_HOME_NETWORK_NAME={{ HOME_NAME }}
+REACT_APP_FOREIGN_NETWORK_NAME={{ FOREIGN_NAME }}
+
+REACT_APP_HOME_WITHOUT_EVENTS={{ HOME_WITHOUT_EVENTS }}
+REACT_APP_FOREIGN_WITHOUT_EVENTS={{ FOREIGN_WITHOUT_EVENTS }}
+
+REACT_APP_HOME_EXPLORER_TX_TEMPLATE={{ UI_HOME_EXPLORER_TX_TEMPLATE }}
+REACT_APP_FOREIGN_EXPLORER_TX_TEMPLATE={{ UI_FOREIGN_EXPLORER_TX_TEMPLATE }}
+REACT_APP_HOME_EXPLORER_ADDRESS_TEMPLATE={{ UI_HOME_EXPLORER_ADDRESS_TEMPLATE }}
+REACT_APP_FOREIGN_EXPLORER_ADDRESS_TEMPLATE={{ UI_FOREIGN_EXPLORER_ADDRESS_TEMPLATE }}
+
+{% if HOME_GAS_PRICE_ORACLE_URL | default('') != '' %}
+REACT_APP_HOME_GAS_PRICE_ORACLE_URL={{ HOME_GAS_PRICE_ORACLE_URL }}
+{% endif %}
+REACT_APP_HOME_GAS_PRICE_SPEED_TYPE={{ HOME_GAS_PRICE_SPEED_TYPE }}
```

Comment:
`REACT_APP_HOME_GAS_PRICE_SPEED_TYPE` also makes sense to move under `if` condition since it is related to the way how data received from the gas price oracle processed.

Reply:
Gotcha - updated as suggested.